ckanext/aircan_connector/action.py

```diff
@@ -27,17 +27,20 @@
                         ' See <a href="https://github.com/datopian/ckanext-aircan#airflow-instance-on-google-composer"> Airflow instance on Google Composer </a>' \
                         ' section in AirCan docs for more.'
 
-def datapusher_submit(context, data_dict):
+
+def aircan_submit(context, data_dict):
     log.info("Submitting resource via Aircan")
     try:
-        res_id = data_dict['resource_id']
+        res_id = data_dict['id']
```

Comment:
@hannelita there is no `id` in `data_dict`, but the `resource_id` is.
should be res_id = data_dict['resource_id']